global.json

```diff
@@ -1,6 +1,9 @@
 {
+  "sdk": {
+    "version": "6.0.300-preview"
+  },
```

Comment:
Do we actually want this? If you're building MAUI, we should trust you know how to select the .NET you want?

Like I normally change `%PATH%`, depending on what I'm doing.

Reply:
> Like I normally change %PATH%, depending on what I'm doing.

We don't really have any [documentation](https://github.com/dotnet/maui/blob/main/.github/DEVELOPMENT.md#running) surrounding this presently. The case can be made to add documentation to do this (on a per platform level). However, I personally prefer this approach as it enforces you're running on the "correct" dotnet version. And this gets updated locally upon `git pull` as MAUI officially starts supporting new versions of the SDK. Currently, again in my experience, it tends to be a bit hit or miss on whether the newest preview is supported or not. For a while I was using 6.0.300* when we were still technically only supporting 6.0.200*, this lead to a good deal of pain when things randomly didn't work. Having this here also helps serve as a single source of truth of what version of the .NET SDK you _should_ be running against.